**This template is provided as a guide to your Jupyter notebook report. You should edit all sections as you see fit. Don't forget to replace the title too!**

# Your Data Mining Pilot Study and Plan

## Background

The New York Times is one of the world’s most circulated newspapers with a history dating back to 1851. A lot has changed in society during the 150 plus year history of The Times, but few things have gained as much attention as the portrayal of women in media. Ever since the recent gender equality discussion (Carlsen, A., Salam, M., Miller, C.C., Lu, D. Ngu, A. Patel, J.K., & Wichter, Z., 2018) change seems to have sped up. It's become increasingly important for newspapers to be mindful of the way they represent women, in order to stay relevant and, perhaps most importantly, not suffer backlash resulting in damage to their image and credibility.

## Problem description

With the recent surge in coverage of media and its treatment of women, the New York Times is interested in looking at the optics of their own publications as it relates to gender. Looking back through history, they are interested in their difference in coverage of male and female subjects, as well as how this relates to the gender of the author.
Our first inquiry relates to how journalists affect the content and coverage that they deliver. According to a study by Armstrong, “the gender of the newswriter can be linked to gender portrayals within news stories.” (Armstrong, C.L., 2008) It suggests that the gender of selected sources and individuals that get covered is related to that of the journalist. Looking at NYT’s publications, we will examine if such a correlation can be found and how this might affect hiring decisions in the future.
Our second question pertains to the representation of women subjects in articles and other news items. According to Collins, Rebecca L. (Sex Roles; New York Vol. 64, Iss. 3-4) women are under-represented across a range of media and settings, including newspapers. Secondly when they are portrayed it is often in a negative manner. We want to research how the representation of women, in articles published by New York Times, has changed over the last 150 years. By doing so we can help NYT understand where they stand right now as it relates to gender representation, and how they have progressed over the years.

## Data collection

*Document your data collection process and the properties of the data here. Implement, using Python code, to load and preprocess your selected dataset.*

*Replace the contents of this cell with your own text.*

In [37]:
import pandas as pd
import numpy as np
%matplotlib inline
import matplotlib.pyplot as plots
import warnings
warnings.simplefilter(action='ignore', category=FutureWarning)
from urllib.request import urlopen
import re

In [43]:
nyt_data = pd.read_csv("C:/Users/johnf/Data Mining And Data Science/Data-Mining-and-Data-Science-Projekt/NYT_data_test.csv", header=None)
nyt_data


C:\Users\johnf\Anaconda3\lib\site-packages\IPython\core\interactiveshell.py:2785: DtypeWarning: Columns (4,8,12,16,20,24,28,32,41,44,49,51,53,56,57,59,61,63,65,67,69,71,73,75,80,83,87,91,95,100,102,103,105,106,107,109,110,119,123,126,132,134,135,138,142,146,150,154,158,162,166,170,174,178,182,192,195,197,199,200,204,207,211,212,213,214,215,216,217,218,219,223,227,231,235,239,240,241,242,243,244,245,246,247,248,249,250,251,252,253,254,255,256,257,258,259,260,261,262,263,264,265,266,267) have mixed types. Specify dtype option on import or set low_memory=False.
  interactivity=interactivity, compiler=compiler, result=result)


0    \
0                                               web_url   
1     https://www.nytimes.com/2016/11/02/world/ameri...   
2     https://www.nytimes.com/2016/11/02/nyregion/ju...   
3     https://www.nytimes.com/2016/11/02/arts/design...   
4     https://www.nytimes.com/2016/11/02/business/de...   
5     https://www.nytimes.com/2016/11/02/world/afric...   
6     https://www.nytimes.com/2016/11/01/well/live/i...   
7     https://www.nytimes.com/2016/11/06/theater/jas...   
8     https://www.nytimes.com/2016/11/06/theater/pla...   
9     https://www.nytimes.com/2016/11/02/sports/ncaa...   
10    https://www.nytimes.com/2016/11/01/sports/worl...   
11    https://www.nytimes.com/2016/11/02/business/in...   
12    https://www.nytimes.com/2016/11/02/business/in...   
13    https://www.nytimes.com/2016/11/02/nyregion/ci...   
14    https://www.nytimes.com/2016/11/02/nyregion/do...   
15    https://www.nytimes.com/2016/11/03/fashion/wha...   
16    https://www.nytimes.com/2016/11/02/business/da...   
17    https://www.nytimes.com/2016/11/03/fashion/isl...   
18    https://www.nytimes.com/video/us/1000000047428...   
19    https://www.nytimes.com/2016/11/02/business/in...   
20    https://www.nytimes.com/2016/11/02/arts/televi...   
21    https://www.nytimes.com/2016/11/02/technology/...   
22    https://www.nytimes.com/interactive/2016/11/01...   
23    https://www.nytimes.com/2016/11/01/briefing/cl...   
24    https://www.nytimes.com/2016/11/02/theater/rev...   
25    https://www.nytimes.com/2016/11/02/arts/design...   
26    https://www.nytimes.com/2016/11/02/briefing/as...   
27    https://news.blogs.nytimes.com/2016/11/01/what...   
28    https://www.nytimes.com/interactive/2016/11/01...   
29    https://www.nytimes.com/2016/11/02/world/europ...   
...                                                 ...   
5389  https://www.nytimes.com/2016/11/29/briefing/eu...   
5390  https://www.nytimes.com/2016/11/29/sports/ohio...   
5391  https://www.nytimes.com/2016/11/29/pageoneplus...   
5392  https://www.nytimes.com/2016/11/29/books/bruce...   
5393  https://www.nytimes.com/2016/11/29/movies/al-b...   
5394  https://www.nytimes.com/2016/11/29/crosswords/...   
5395  https://www.nytimes.com/video/business/1000000...   
5396  https://www.nytimes.com/2016/11/29/us/north-ca...   
5397  https://www.nytimes.com/2016/11/29/nyregion/do...   
5398  https://www.nytimes.com/2016/11/29/us/tom-pric...   
5399  https://www.nytimes.com/2016/11/29/arts/music/...   
5400  https://www.nytimes.com/2016/11/29/nyregion/bi...   
5401  https://www.nytimes.com/2016/11/29/sports/foot...   
5402  https://www.nytimes.com/2016/11/29/nyregion/lo...   
5403  https://www.nytimes.com/2016/11/29/world/canad...   
5404  https://www.nytimes.com/2016/11/29/us/politics...   
5405  https://www.nytimes.com/2016/11/29/sports/base...   
5406  https://www.nytimes.com/2016/11/29/nyregion/be...   
5407  https://www.nytimes.com/2016/11/29/us/charlest...   
5408  https://www.nytimes.com/2016/11/29/opinion/mr-...   
5409  https://www.nytimes.com/2016/11/29/business/me...   
5410  https://www.nytimes.com/2016/11/29/nyregion/br...   
5411  https://www.nytimes.com/2016/11/29/todayspaper...   
5412  https://www.nytimes.com/2016/11/29/sports/socc...   
5413  https://www.nytimes.com/2016/11/29/nyregion/me...   
5414  https://www.nytimes.com/2016/11/29/opinion/in-...   
5415  https://www.nytimes.com/2016/11/29/business/tr...   
5416  https://www.nytimes.com/2016/11/29/world/middl...   
5417  https://www.nytimes.com/2016/11/29/sports/golf...   
5418  https://www.nytimes.com/2016/11/29/fashion/sam...   

                                                    1    \
0                                               snippet   
1     Investigators looking into corruption within t...   
2     Lawyers for Bridget Anne Kelly and Bill Baroni...   
3     The estate mostly prevailed in a legal dispute...   
4     The bank will reimburse more than 250,000 cust...   
5     An investigation found that a pea

## Data analysis

*Document you choice and motivation for selected data mining method(s) here. Choose a data mining method(s) to use in Python code to perform an analysis of your chosen dataset. Describe why you chose the method(s) and what interesting things you have found from the analysis.*

*Replace the contents of this cell with your own text.*

In [3]:
# Add your own code

## Evaluation of results

*Document an evaluation your analysis results and describe how potentially actionable they are.*

*Replace the contents of this cell with your own text.*

In [ ]:
# Add your own code

## Schedule and description of project plan

*Rough schedule for the project beyond the pilot study presented in 3-5. This does not have to be advanced, you can simply provide an estimate based upon reported schedules for similar projects in the literature.*

*Replace the contents of this cell with your own text.*

## Ethical aspects that need to be considered

*Are there ethical aspects that need to be considered? Are there legal implications (e.g., personal data / GDPR)? Are there implications if the case organization is a business, public authority, or nonprofit entity?*

*Replace the contents of this cell with your own text.*

## References

Armstrong, C. L. (2004) ‘The Influence of Reporter Gender on Source Selection in Newspaper Stories’, Journalism & Mass Communication Quarterly, 81(1), pp. 139–154. Available at: https://search-ebscohost-com.ezproxy.its.uu.se/login.aspx?direct=true&db=eue&AN=507903756&site=ehost-live (Accessed: 13 November 2018).

Carlsen, A., Salam, M., Miller, C.C., Lu, D. Ngu, A. Patel, J.K., & Wichter, Z. (2018) ‘#MeToo Brought Down 201 Powerful Men. Nearly Half of Their Replacements Are Women.’ The New York Times, updated 19th Oct [Online]. Available: https://www.nytimes.com/interactive/2018/10/23/us/metoo-replacements.html (Accessed on 2018-11-12)

Collins, R. L. (2011). Content analysis of gender roles in media: Where are we now and where should we go? Sex Roles, 64(3-4), 290-298. doi:http://dx.doi.org.ezproxy.its.uu.se/10.1007/s11199-010-9929-5